src/workers/jobs.js

```diff
@@ -169,16 +168,15 @@ export async function assignTexters(job) {
     }
     await updateJob(job, Math.floor((75 / texterCount) * (index + 1)) + 20)
   }
-  const assignmentsToDelete = await r.table('assignment')
-    .getAll(id, { index: 'campaign_id' })
-    .merge((row) => ({
-      count: r.table('campaign_contact')
-        .getAll(row('id'), { index: 'assignment_id' })
-        .count()
-    }))
-    .filter((row) => row('count').eq(0))
+  const assignmentsToDelete = await r.knex.from('assignment')
+    .where('campaign_id', id)
+    .join('campaign_contact', 'id', 'assignment_id')
+    .groupBy('assignment_id')
+    .select('assignment_id')
+    .havingRaw('COUNT(campaign_contact.id) = 0')
```

Comment:
i'm just doing what the app did before.  If we want to archive assignments, then let's make that a github issue to prioritize, etc.